### PR TITLE
pipewire: 0.3.24 -> 0.3.25

### PR DESCRIPTION
**Pull request by jansol**
###### Motivation for this change

https://gitlab.freedesktop.org/pipewire/pipewire/-/releases/0.3.25

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/libraries/pipewire/default.nix [hunk 1]
```diff
@@ -75,11 +75,6 @@ let
       ./0070-installed-tests-path.patch
       # Add flag to specify configuration directory (different from the installation directory).
       ./0080-pipewire-config-dir.patch
-      # Fix JSON parser.
-      (fetchpatch {
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/pipewire/default.nix:79]: Please remove fetchpatch from inputs.

**Comment by Mic92** [comment]
Do we need to update json files in `nixos/modules/services/desktops/pipewire`?

**Comment by lovesegfault** [comment]
@alyssais Would you like to test this before it's merged?

**Comment by ashkitten** [comment]
working well for me

**Comment by lovesegfault** [comment]
Working here too

**Pull request merged**